src/vs/workbench/contrib/dialogs/browser/dialogHandler.ts

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 import * as nls from 'vs/nls';
```

Comment:
I see a NLS warning here

Reply:
i don't see this in editor or the watch task

Reply:
@sbatten please make sure you have setup vscode properly to see these warnings, todays build is broken: https://dev.azure.com/vscode/VSCode/_build/results?buildId=57514&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=3367cfe7-6aa5-5a88-c888-e6814d74f6eb&l=25